client/src/dialogs/Analytics/AnalyticsDialog.tsx

```diff
@@ -151,7 +151,6 @@ function AnalyticsDialog (): React.ReactElement {
             <div className="dialog-actions">
               <CapacitySources />
               <Checkbox
-                id="show-analytics"
```

Comment:
There remain a number of these files from a previous PR (https://github.com/streetmix/streetmix/pull/3180) that shouldn't be in this diff. You should be able to resolve this and clean up the PR by rebasing your changes on the current `main` branch, and it would be helpful to squash your commits to one. This would improve the diff and make it easier to review and as a record for the future. Please address this and let me know if you need any help with this. Thanks!

Reply:
sure i will solve this.

Reply:
Hmm, this should no longer appear in the diff. Are you absolutely certain you have rebased your changes on the latest `main` branch? If so, you may need to manually remove these files from your commit.